docs/setup.md

```diff
@@ -395,16 +395,13 @@ supported.
     sudo chown paperless:paperless /opt/paperless/consume
     ```
 
-8.  Install python requirements from the `requirements.txt` file. It is
-    up to you if you wish to use a virtual environment or not. First you
-    should update your pip, so it gets the actual packages.
+8.  Install python requirements from the `Pipfile` file using `pipenv`. It is
```

Comment:
`requirements.txt is still provided in the release package and is preferred over users needing to install `pipenv`

Reply:
This change also needs to be addressed still.

Reply:
You're right. I forgot about the changes made. I'll just revert them at this point.